domain/status/package_test.go

```diff
@@ -4,3 +4,4 @@
 package status
 
 //go:generate go run go.uber.org/mock/mockgen -typed -package status_test -destination lease_mock_test.go github.com/juju/juju/core/lease LeaseManager
+//go:generate go run go.uber.org/mock/mockgen -typed -package status_test -destination leadership_mock_test.go github.com/juju/juju/domain/status/service ControllerState
```

Comment:
This isn't needed? This mock is being generated for the top level domain package. The generate commands that already exist in the service make the mocks in that domain for the ControllerState.